### ISSUE TITLE
await is a unary expression now

### ISSUE THREAD

**Issue by gabelevi** [benjamn/ast-types#113]
Async/await now has a [semi-official posted draft spec](http://tc39.github.io/ecmascript-asyncawait/#UnaryExpression) and they've decided to make `await foo` a unary expression.

Any opinion on how to move forward? Support definitions for both `AwaitExpression` and add `await` to `UnaryOperator`s in es7.js?

CC @nmote

**Comment by benjamn** [comment]
Yep, I think that's reasonable. In particular, there is no immediate need to deprecate `AwaitExpression`. This library is designed so that AwaitExpression and `await`-as-`UnaryOperator` can coexist in the type space for as long as both are useful.
